deepcell/model_zoo/fpn.py

```diff
@@ -79,14 +82,22 @@ def create_pyramid_level(backbone_input,
         ValueError: ndim is not 2 or 3
         ValueError: upsample_type not ['upsamplelike','upsampling2d','upsampling3d']
     """
-
+    # Check input to ndims
     acceptable_ndims = {2, 3}
     if ndim not in acceptable_ndims:
         raise ValueError('Only 2 and 3 dimensional networks are supported')
 
+    # Check if inputs to ndim and lite are compatible
     if ndim == 3 and lite:
         raise ValueError('lite == True is not compatible with 3 dimensional networks')
 
+    # Check input to interpolation
+    acceptable_interpolation = {'bilinear', 'nearest'}
+    if interpolation not in acceptable_interpolation:
+        raise ValueError('Interpolation mode not supported. Choose from '
+            '[\'bilinear\', \'nearest\']')
```

Comment:
The strings have to align to pass PEP8, also I suggest using double-quotes inside the string for better readability.

```
        raise ValueError('Interpolation mode not supported. Choose from '
                                   '["bilinear", "nearest"]')
```